test/images/pets/zookeeper-installer/install.sh

```diff
@@ -55,10 +55,10 @@ cp "${INSTALL_VOLUME}"/zookeeper/conf/zoo_sample.cfg "${INSTALL_VOLUME}"/zookeep
 
 # TODO: Should dynamic config be tied to the version?
 IFS="." read -ra RELEASE <<< "${VERSION}"
-if [ $(expr "${RELEASE[1]}") -gt 4 ]; then
+if [ "$(("${RELEASE[1]}"))" -gt 4 ]; then
     echo zookeeper-"${VERSION}" supports dynamic reconfiguration, enabling it
     echo "standaloneEnabled=false" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
-    echo "dynamicConfigFile="${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg.dynamic" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
+    echo "dynamicConfigFile=${INSTALL_VOLUME}/zookeeper/conf/zoo.cfg.dynamic" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
```

Comment:
Why I do this change:

```
In test/images/pets/zookeeper-installer/install.sh line 61:
    echo "dynamicConfigFile="${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg.dynamic" >> "${INSTALL_VOLUME}"/zookeeper/conf/zoo.cfg
                             ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                             ^-- SC2086: Double quote to prevent globbing and word splitting.
```